### ISSUE TITLE
Allow the forced installation of packages in environment with specification conflicts

### ISSUE THREAD

**Issue by jgmorrison** [conda/conda#4085]
Customers working with Conda environments can no longer use "--force" or "--no-deps" to create an environment with specification conflicts. This worked in Conda-3.x versions but no longer in Conda-4.x now that the solver considers the environment. Customers may need this in situations where they have vendor packages that are known to work with any version of a package but their specifications do not correctly state it(Vendor package is known to work with any version of Numpy above 1.6 but the vendor package specifies up to 1.8 while other packages need 1.9). Conda-4.x has caused breaking changes to enterprise customers although this was not necessarily what "--force" was meant for.

Is there a way to add a feature or functionality that will allow users to create an environment with conflicting specification if needed? This is seen as lost functionality in their use.

    conda install bokeh=0.9.2 numpy=1.8 --force

    conda install bokeh=0.9.2 numpy=1.8 --no-deps

**Comment by kalefranz** [comment]
This might be behavior I've fixed in 4.3.  But, with conda 4.3.1, I was able to do the following

Save this file to `bokeh.explicit.txt`
```
# This file may be used to create an environment using:
# $ conda create --name <env> --file <this file>
# platform: osx-64
@EXPLICIT
https://repo.continuum.io/pkgs/free/osx-64/backports-1.0-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/backports_abc-0.5-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/bokeh-0.9.2-np19py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/colorama-0.3.7-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/flask-0.10.1-py27_2.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/gevent-1.0.1-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/gevent-websocket-0.9.3-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/greenlet-0.4.11-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/itsdangerous-0.24-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/jinja2-2.8-py27_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/markupsafe-0.23-py27_2.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/mkl-11.3.3-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/numpy-1.8.2-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/openssl-1.0.2j-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pip-9.0.1-py27_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pygments-2.1.3-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/python-2.7.13-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/python-dateutil-2.6.0-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pytz-2016.10-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pyyaml-3.12-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/pyzmq-14.7.0-py27_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/readline-6.2-2.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/requests-2.12.4-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/setuptools-27.2.0-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/singledispatch-3.4.0.3-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/six-1.10.0-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/sqlite-3.13.0-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ssl_match_hostname-3.4.0.2-py27_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/tk-8.5.18-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/tornado-4.4.2-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/ujson-1.33-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/werkzeug-0.11.11-py27_1.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/wheel-0.29.0-py27_0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/yaml-0.1.6-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/zeromq-4.1.3-0.tar.bz2
https://repo.continuum.io/pkgs/free/osx-64/zlib-1.2.8-3.tar.bz2
```

Notice `bokeh-0.9.2-np19py27_0` vs `numpy-1.8.2-py27_0`.

The command

    conda create --name broken-env --file bokeh.explicit.txt

worked for me without issue.

Obviously, these are full URLs to tarball files.  If the files are exist locally rather than a remote server, something like `file:///path/to/my/package/numpy-1.8.2-py27_0.tar.bz2` should work.

**Comment by kalefranz** [comment]
Having no additional feedback on this issue, moving to close.
